### PR TITLE
Issue 4590 fab of custom selector is still open after uploading

### LINKED ISSUE #4590: FAB of custom selector is still open after uploading
**Summary:** 

FAB of the custom selector is still open after uploading an image from it and taking the user to the last opened folder

**Steps to reproduce:** 

1. Click + icon and click FAB of the custom selector
2. upload an image

What did you expect the app to do, and what did you see instead?

It should close the FAB after clicking it.

**Device and Android version:** 

Pixel 2 API 30
 
**Commons app version:** 

Betadebug latest master

### PR DESCRIPTION
**Pull request by jmora13**
**Description (required)**
Fab list was still open after uploading photo through custom selector fab. It should be closed upon returning to ContributorListFragment.

Fixes #4590

What changes did you make and why?
Called the function which animates (open / closes) the fab list after the custom selector fab was clicked.

**Tests performed (required)**
Tested betaDebug on Pixel 4 with API level 30.

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java [hunk 1]
```diff
@@ -169,15 +173,17 @@ private void prepareAppLanguages(final String keyListPreference) {
             Objects.requireNonNull(getContext()));
         languageNamesList = appLanguageLookUpTable.getLocalizedNames();
         languageCodesList = appLanguageLookUpTable.getCodes();
-        List<String> languageNameWithCodeList = new ArrayList<>();
+        final SortedMap<String, String> languageMap = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java:176]: I guess this is not related to #4590? :-)

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]

File: app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java (hunk 1 above)
Comment (Ayan-10) [reply app/src/main/java/fr/free/nrw/commons/settings/SettingsFragment.java:176]: This code is related to issue #4502 and PR #4623 . 

@jmora13 For solving each issue create a different branch from **master** to avoid this kind of overlapping.
